src/serialization/index.ts

```diff
@@ -51,4 +51,5 @@ export {
   Uint8List,
   VoidList,
   Void,
+  type _Pointer,
```

Comment:
Without this, some generated `.ts` files included a `_capnp` property that referenced a `_Pointer` that TS couldn't find, causing:

```
error TS4026: Public static property '_capnp' of exported class has or is using name '_Pointer' from external module but cannot be named
```

Reply:
can we have a test perhaps to avoid regression?